macros/src/derive_fn.rs

```diff
@@ -121,19 +133,25 @@ pub fn derive_fn(
     let spec_ident = format_ident!("__SPEC_XDR_{}", ident.to_string().to_uppercase());
 
     // Generated code.
-    quote! {
-        #[cfg_attr(target_family = "wasm", link_section = "contractspecv0")]
-        pub static #spec_ident: [u8; #spec_xdr_len] = *#spec_xdr_lit;
+    if let Some(cfg_feature) = feature {
+        quote! {
+            #[cfg_attr(target_family = "wasm", link_section = "contractspecv0")]
+            pub static #spec_ident: [u8; #spec_xdr_len] = *#spec_xdr_lit;
 
-        #[export_name = #wrap_export_name]
-        pub fn #wrap_ident(env: stellar_contract_sdk::Env, #(#wrap_args),*) -> stellar_contract_sdk::RawVal {
-            <_ as stellar_contract_sdk::IntoVal<stellar_contract_sdk::Env, stellar_contract_sdk::RawVal>>::into_val(
-                #call(
-                    #env_call
-                    #(#wrap_calls),*
-                ),
-                &env
-            )
+            #[cfg(feature = #cfg_feature)]
+            #[export_name = #wrap_export_name]
```

Comment:
I think you could probably reuse much of this block and substitute in just the export part instead, and then wrap the export_name in a cfg_attr:
```suggestion
            #[cfg_attr(feature = #cfg_feature, export_name = #wrap_export_name)]
```

Reply:
I was not aware of `cfg_attr`, that's neat. I'll update this which should make the PR quite a bit simpler.